playwright/src/main/java/com/microsoft/playwright/BrowserContext.java

```diff
@@ -127,6 +127,16 @@ public interface BrowserContext extends AutoCloseable {
    */
   void offPage(Consumer<Page> handler);
 
+  /**
+   * Emitted when unhandled exceptions occur on any pages created through this context. To only listen for {@code pageError}
+   * events from a particular page, use {@link Page#onPageError Page.onPageError()}.
+   */
+  void onPageError(Consumer<PageError> handler);
```

Comment:
the project won't build until these methods are implemented.  I stubbed them out for my purposes but then removed the stubs before creating this PR.

Reply:
I'm rolling the driver in https://github.com/microsoft/playwright-java/pull/1362 with implementation of all the new APIs but SoftAssertions. You'll need to revert my change in ApiGenerator.java to make it produce SoftAssertions.java